terraform/environments/data-platform/src/presigned_url/main.py

```diff
@@ -0,0 +1,43 @@
+import os
+import json
+import boto3
+import logging
+from datetime import datetime
+
+logging.basicConfig(level=logging.INFO, force=True)
+root_logger = logging.getLogger()
+s3 = boto3.client("s3")
+
+
+def handler(event, context):
+    bucket_name = os.environ["bucketname"]
+    amz_date = datetime.utcnow().strftime("%Y%m%dT%H%M%SZ")
+    file_name = event["queryStringParameters"]["filename"]
```

Comment:
Everything seems ok , regarding security perspective. 
Can you check the possibility of implementing random UUID for file name point (6) and signed headers (80. https://insecurity.blog/2021/03/06/securing-amazon-s3-presigned-urls/

Reply:
would we have to keep track of the file extension for parsing the file when the name is replaced by a UUID? eg csv or json 

renaming is fine if we expect one predictable input type